src/common/darktable.c

```diff
@@ -770,6 +770,30 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
     gtk_init(&argc, &argv);
   }
 
+  // execute a performance check and configuration if needed
+  int last_configure_version = dt_conf_get_int("performance_configuration_version_completed");
+  if(last_configure_version < DT_CURRENT_PERFORMANCE_CONFIGURE_VERSION)
+  {
+    // if this is the very first time -> run this configuration unconditionally
```

Comment:
The comment is no longer true. And you don't need to initialize the variable either. scan-build will complain about that.